static/app/views/replays/detail/trace.tsx

```diff
@@ -108,12 +109,25 @@ export default function Trace({replayRecord, organization}: Props) {
           )
         );
 
+        const successfulTraceDetails = traceDetails
+          .map(settled => (settled.status === 'fulfilled' ? settled.value[0] : undefined))
+          .filter(Boolean);
+
+        if (successfulTraceDetails.length !== traceDetails.length) {
+          traceDetails.forEach(trace => {
+            if (trace.status === 'rejected') {
+              Sentry.captureMessage(trace.reason);
+            }
+          });
+        }
```

Comment:
I noticed this transaction was throwing a 404, it doesn't want to load.

https://dev.getsentry.net:7999/organizations/sentry/performance/trace/76d6d097c07b498b9d2d183eae9bfbd5/?pageEnd=2022-08-20T07%3A31%3A57.069&pageStart=2022-08-19T07%3A31%3A57.06

<img width="1072" alt="Screen Shot 2022-08-19 at 2 33 01 PM" src="https://user-images.githubusercontent.com/187460/185710963-88acf124-dde6-43a8-ad33-af2905159e49.png">
9

So I decided to try `allSettled` so that we could see the rest of the transaction data, and log that something bad happened here.

Reply:
The frontend TSC never really made a decision for what we want to support, but support by date released seems to be relatively good for the last 3 years. Ideally we could track this with our actual browser distribution...

![image](https://user-images.githubusercontent.com/79684/185715211-6127a3ba-4ea7-4da9-aacc-dfaef9ae2c05.png)